cloudbuild/cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
```

Comment:
What's the reasoning for duplicating this in the repo?

It seems that the Cloud Build samples and docs mention storing this stuff in the repo, but that seems to conflict with our community's normal approach of using external dependencies instead of copying code, so I'd love to understand why, and if it's required.

I'd love to have something like a 1 button deploy (ala Heroku) for the docker images so developers can just deploy the image to their environment straight from the site and configure their yaml files accordingly.

Reply:
No idea, I was mainly just copying stuff in so that I can hack on things.

I made a few tweaks to the Android image, just to remove things I don't need (NDK, etc).

You could expose these all as separate repos, and then use hub.docker.com to build/host them. That way anyone can (theoretically) use them.